examples/bert/run_pretraining.py

```diff
@@ -142,18 +142,6 @@ def call(self, features: tf.Tensor):
         x = self.out_proj(x)
         return x
 
-    def get_config(self):
```

Comment:
Why aare we deleting the `get_config` here?

Reply:
The head is never saved, and we don't want to expose it as a component to other scripts. So I think this does nothing?

Checkpointing doesn't need it. Let's keep things minimal.